keras_hub/src/models/retinanet/retinanet_image_converter.py

```diff
@@ -6,16 +6,3 @@
 @keras_hub_export("keras_hub.layers.RetinaNetImageConverter")
 class RetinaNetImageConverter(ImageConverter):
     backbone_cls = RetinaNetBackbone
-
-    def __init__(
-        self,
-        *args,
-        **kwargs,
-    ):
-        # TODO: update presets and remove these old config options. They were
```

Comment:
Have we already updated the preset files? We don't need these shims anymore?

Reply:
Not yet, but with this we can update the files, I will do that

Reply:
Probably good practice to ditch this change from this PR then. And add this change to the upcoming PR that updates the presets.

Reply:
I will update the presets as well as script is ready and I already generated presets just the uploading to Kaggle is pending

Reply:
Ok! I'll just wait for the preset update on this PR then, either way fine. (but let's avoid removing the compat hack before we add the new presets)

Reply:
Uploaded new weights version in Kaggle and updated the presets.